app/assets/javascripts/components/components/media_gallery.jsx

```diff
@@ -160,6 +160,8 @@ const Item = React.createClass({
     } else if (attachment.get('type') === 'gifv') {
       thumbnail = (
         <video
+          tabIndex='0'
+          role='button'
```

Comment:
This is a false-positive of the linter. Fixed in https://github.com/evcohen/eslint-plugin-jsx-a11y/pull/194

We don't want to turn videos into buttons :) It doesn't need the tabIndex or the role attributes.